tests/RME/gold_hrrr/config.ini

```diff
@@ -16,8 +16,9 @@
 [topo]
 basin_lon:                     -116.7547 
 basin_lat:                     43.067    
-filename:                      /home/micahsandusky/Code/smrf/tests/RME/topo/topo.nc
+filename:                      /home/scotthavens/code/smrf/tests/RME/topo/topo.nc
```

Comment:
Lets changes the paths back to relative to avoid user names.

Reply:
Removing